projects/core/src/cart/store/effects/cart.effect.spec.ts

```diff
@@ -10,7 +10,7 @@ import { cold, hot } from 'jasmine-marbles';
 
 import { CartConnector } from '../../connectors/cart/cart.connector';
 import * as fromActions from '../actions/cart.action';
-import { ProductImageNormalizer } from '../../../product';
+import { ProductImageNormalizer } from '../../../occ/adapters/product/converters/product-image-normalizer';
```

Comment:
do we need to use the `ProductImageNormalizer` here anyway? Would be cleaner to keep this class with its tests in the occ folder

Reply:
Yes, you are right, fixed that